src/Phinx/Db/Adapter/BambooHRMysqlAdapter.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Phinx\Db\Adapter;
+
+class BambooHRMysqlAdapter extends \Phinx\Db\Adapter\MysqlAdapter {
+
+	public function hasTable($tableName) {
+		$sql = sprintf("DESC %s", $tableName);
+		try {
+			$exists = $this->query($sql);
+		} catch (\PDOException $exc) {
+			return false;
+		}
+		return true;
+	}
+
+	protected function getForeignKeys($tableName) {
+		throw new \Exception("uh uh - we aren't going to support getting foreign keys");
+	}
```

Comment:
If we are confident that the above implementation will work all the time (or we had testing for it), I wouldn't mind. What would this enable in a migration that we really want?